definitions/tools/star_align.cwl

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env cwl-runner
+
+class: CommandLineTool
+cwlVersion: v1.0
+label: "STAR: align reads to transcriptome"
+baseCommand: STAR
+requirements:
+    - class: ShellCommandRequirement
+    - class: ResourceRequirement
+      ramMin: 32000
+      coresMin: 16
+    - class: DockerRequirement
+      dockerPull: "sridnona/mgirnaseq:v1"
+arguments: [
```

Comment:
I'll suggest that we push this to mgibio if it's a dockerfile you've created. Just helpful to keep core pipeline dockers all in one place!